addon.py

```
@@ -1373,6 +1387,311 @@ def import_generated_asset_fal_ai(self, request_id: str, name: str):
+                    f.write(chunk)
+
+            # Unzip the ZIP
+            with zipfile.ZipFile(zip_file_path, "r") as zip_ref:
+                zip_ref.extractall(temp_dir)
+
+            # Find the .obj file (there may be multiple, assuming the main file is model.obj)
+            for file in os.listdir(temp_dir):
+                if file.endswith(".obj"):
+                    obj_file_path = osp.join(temp_dir, file)
+                elif file.endswith(".mtl"):
+                    mtl_file_path = osp.join(temp_dir, file)
```

Ruff (0.11.9)

Local variable `mtl_file_path` is assigned to but never used

Remove assignment to unused variable `mtl_file_path`

(F841)